### PR TITLE
Payment

### PR DESCRIPTION
**Pull request by Djmiller1717**
Integrated Stripe for checkout.

Create a post route on stripe.js and a put route on the order.
Cart component updated to have "Keep Shopping" and "Checkout" button work accordingly.
Stripe Checkout component created.
Successful payment component created as well.
Put Route tripped upon successful payment to update isPaid property to true.

### CODE REVIEW

File: client/components/Cart.js [hunk 1]
```diff
@@ -54,8 +77,8 @@ class Cart extends Component {
           $
           { cartTotal }
         </h2>
-        <button>Check Out Now</button>
-        <button>Keep Shopping</button>
+        <button type="button" onClick={(ev) => this.checkout(ev)}>Check Out Now</button>
+        <button type="button" onClick={(ev) => this.keepShopping(ev)}>Keep Shopping</button>
```
Comment (fugoto) [inline client/components/Cart.js]: these can be just {this.checkout}, {this.keepshopping}

File: client/components/Payment/PaymentSuccess.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  keepShopping(ev) {
+    ev.preventDefault();
+    this.props.history.push('/Home');
+  }
+
+  render() {
+    return (
+      <div>
+        <div>You payment was processed!</div>
+        <button type="button" onClick={(ev) => this.keepShopping(ev)}>Continue Shopping</button>
```
Comment (fugoto) [inline client/components/Payment/PaymentSuccess.js]: these can be just {this.keepshopping}

**Review by fugoto — CHANGES_REQUESTED** [review]
thanks again for this! i tested and i think it looks good. I think one thing is that on the back end, shipping address, email etc would need to also be updated in the database with the info that user puts in on stripe - probably in same route as where isPaid is set to true? I also did test that (ev)...is not needed on some of the onClicks, I commented in those places. let me know if you have any questions!

File: client/components/Cart.js (hunk 1 above)
Comment (Djmiller1717) [reply client/components/Cart.js]: updated.

File: client/components/Payment/PaymentSuccess.js (hunk 2 above)
Comment (Djmiller1717) [reply client/components/Payment/PaymentSuccess.js]: updated.

**Review by fugoto — APPROVED** [review]

**Pull request merged**